conf/bootstrap.coffee

```diff
@@ -1,31 +1,30 @@
 
 module.exports =
-  '@nikitajs/core/lib/log/cli':
+  cli:
     pad: host: 20, header: 60
-  '@nikitajs/core/lib/log/md':
+  log:
     basedir: "#{__dirname}/../log"
-  '@nikitajs/core/lib/ssh/open':
-    disabled: true
```

Comment:
This will be required, comment it but don't remove it.

Reply:
I might have missed something but the way it was written before with the loop that call all actions present in their respective JS objects was not working. Nikita was exiting without message beside having the right namespace.

That is why I change the `run.coffee` file and the way I call the actions.

Reply:
It was working before. Maybe not with the lastest version but we should make it work